### ISSUE TITLE
Autofill/tokenize usernames from /chat-names

### ISSUE THREAD

**Issue by EdwardHinkle** [indieweb/chat.indieweb.org#49]
One thing that is really nice in Slack (especially in Slack's that aren't bridged with IRC) is being able to start typing and autofill a nickname of someone you want to mention.

It would be great if the [/chat-names](https://indieweb.org/chat-names) were logged and we had a javascript autocomplete UI that allowed you to autocomplete a username. This would allow you to more easily log !tells and such functions when trying to mention someone.

**Comment by Cambridgeport90** [comment]
Yes. This would definitely be nice to add... I screw up people's names all the time... and then people wonder why never received a notification that I mentioned them. Happens all the time.
